main/main.c

```diff
@@ -45,6 +46,59 @@ uint64_t cycles_per_sec;
 uint64_t cycles_per_ms;
 uint64_t picosec_per_cycle;
 
+char *lua_base_dir = NULL;
+char *gatekeeper_config_file = NULL;
```

Comment:
1. Can't we add `const`  before `char` for both variables? If so, remember to update the header file `include/gatekeeper_main.h`.

2. Since they are **global** variables, they are automatically initialized to `NULL`, so drop the initialization.